ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -29,12 +29,13 @@
           </v-btn>
         </v-col>
         <v-col style="padding-top: 2px;">
-          <p v-if="isDraft(item)" :class="{ 'ma-0': true, 'pl-9': isChild }">Pending</p>
+          <p v-if="isDraft(item)" :class="{ 'ma-0': true, 'pl-9': isChild}">Pending</p>
```

Comment:
Not a deal breaker but the space can stay, probably a misclick?

Reply:
Made the change...